test/plugins/sqlite.js

```diff
@@ -0,0 +1,102 @@
+"use strict";
+
+const fs = require("fs");
+const path = require("path");
+const expect = require("chai").expect;
+const Msg = require("../../src/models/msg");
+const Helper = require("../../src/helper");
+const MessageStorage = require("../../src/plugins/sqlite.js");
+
+describe("SQLite Message Storage", function() {
+	const expectedPath = path.join(Helper.getHomePath(), "logs", "testUser.sqlite3");
+	let store;
+
+	// Delete database file from previous test run
+	before((done) => {
+		store = new MessageStorage();
+
+		if (fs.existsSync(expectedPath)) {
+			fs.unlink(expectedPath, done);
+		} else {
+			done();
+		}
+	});
+
+	it("should resolve an empty array when disabled", (done) => {
+		store.getMessages(null, null).then((messages) => {
+			expect(messages).to.be.empty;
+			done();
+		});
+	});
+
+	it("should create database file", () => {
+		expect(store.isEnabled).to.be.false;
+		expect(fs.existsSync(expectedPath)).to.be.false;
+
+		store.enable("testUser");
+
+		expect(store.isEnabled).to.be.true;
+		expect(fs.existsSync(expectedPath)).to.be.true;
```

Comment:
You could use `pathExists` here as well.

Reply:
That doesn't work as `pathExists` uses `fs.access`, but the file is locked by sqlite.

Reply:
Argh, that sucks. Oh well...